src/process/redux/projectSlice.js

```diff
@@ -1,11 +1,10 @@
-import axios from 'axios';
 import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 
-import baseUrl from '../api/index';
+import api from '../api/index';
```

Comment:
Import it in here with an uppercase A and use it like this everywhere in here.
Also you don't need to specify `index`.
```suggestion
import api from '../Api';
```